common/changes/@microsoft/rush/master_2019-12-14-00-08.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Add a --commit flag to publish action to specify the git commit hash to tag at when publishing.",
```

Comment:
This might flow nicer with just:
"Add a --commit flag to specify the git commit hash to tag when publishing."

Reply:
sounds good to me.